services/passwordless_token_verifier.go

```diff
@@ -32,6 +32,8 @@ func PasswordlessTokenVerifier(store data.AccountStore, r ops.ErrorReporter, cfg
 		return 0, FieldErrors{{"account", ErrLocked}}
 	} else if account.Archived() {
 		return 0, FieldErrors{{"account", ErrLocked}}
+	} else if account.LastLoginAt.Sub(claims.IssuedAt.Time()) > 0 {
```

Comment:
ahhh, nice, we don't need a `Lock` claim after all. good find!

would `Before` work here?

```suggestion
	} else if account.LastLoginAt.Before(claims.IssuedAt.Time()) {
```

Reply:
Would not `After`? Returns the `ErrInvalidOrExpired` error if exists a login (`LastLoginAt`) after request a passwordless token (`claims.IssuedAt`) ?